ci-operator/step-registry/cucushift/upgrade/check/capability/cucushift-upgrade-check-capability-commands.sh

```diff
@@ -0,0 +1,421 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'FRC=$?; create_postupgrade_junit' EXIT TERM
+
+# Generate the Junit for cap check
+function create_postupgrade_junit() {
+    echo "Generating the Junit for post upgrade check on capability"
+    filename="cluster upgrade"
+    testsuite="cluster upgrade"
+    subteam="OTA"
+    if (( FRC == 0 )); then
+        cat >"${ARTIFACT_DIR}/${filename}.xml" <<EOF
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuite name="${testsuite}" failures="0" errors="0" skipped="0" tests="1" time="1">
+  <testcase name="${subteam}:Post upgrade check on capability should succeed or skip"/>
+</testsuite>
+EOF
+    else
+        cat >"${ARTIFACT_DIR}/${filename}.xml" <<EOF
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuite name="${testsuite}" failures="1" errors="0" skipped="0" tests="1" time="1">
+  <testcase name="${subteam}:Post upgrade check on capability should succeed">
+    <failure message="">Post upgrade check on capability failed</failure>
+  </testcase>
+</testsuite>
+EOF
+    fi
+}
+
+baselinecaps_from_cluster=$(oc get clusterversion version -ojson | jq -r '.spec.capabilities.baselineCapabilitySet')
```

Comment:
Maybe we should get the baseline cap set from install.yaml, and ensure the baseline cap set equals to the value from cvo?

Reply:
`cv.spec.capabilities` reflects the cap we defined in install-config.yaml. It looks like an install check point which I assume passed when we run the step.

Reply:
I checked `ci-operator/step-registry/cucushift/installer/check/capability/cucushift-installer-check-capability-commands.sh`, seem like there is no such check point, @jinyunma do you think we can add the check point in the  installer check script?

Reply:
@shellyyang1989 I am thinking another point, even we added installer check point, shall we also check the setting after the upgrade step completed.

Reply:
Good point, we missed this checkpoint on installer side, it's okay to add it in installer check script.

Reply:
Do you mean if we find the `cv.spec.capabilities` is different from the cap defined in install-config.yaml, we skip the check?

Reply:
I meant is that possible, the installer checkpoint get passed, while after upgrade, the `cv.spec.capabilities` get different with the user setting from the fresh install.

Reply:
IMO, it's a rule in CI to not change the cluster behavior through the test. Even if someone changes some resources but they must ensure those resources are reverted back after their test. It's a dangerous operation to day-2 update cap setting in CI because enabled resources can't be disabled any more. So I'm assuming the cap set is not changed.

Reply:
I did not mean some e2e testing changed the enabled caps, but maybe some CVO bugs cause the unexpected change.

Reply:
ah, got you. `cv.spec.capabilities` reflects the cap set what user specifies, so CVO will set the field based on install-config.yaml in the beginning of installation. Later on, CVO will never update it. CVO only reflects cluster changes to `cv.status`